src/common/snippets/src/lowered/pass/init_registers.cpp

```diff
@@ -23,7 +23,9 @@ bool InitRegisters::run(LinearIR& linear_ir) {
     lowered::pass::PassPipeline reg_pipeline(m_pass_config);
     reg_pipeline.register_pass<lowered::pass::InitLiveRanges>(m_reg_manager);
     reg_pipeline.register_pass<lowered::pass::AssignRegisters>(m_reg_manager);
+#ifdef OPENVINO_ARCH_X86_64
     reg_pipeline.register_pass<lowered::pass::InsertRegSpills>(m_reg_manager);
+#endif
```

Comment:
The same comment - hw specific conditions should not be in common parts of Snippets.

I recommend to just disable this pass using `PassConfig` on [plugin side](https://github.com/chenhu-wang/openvino/blob/8dfba86f2384c74cc010c68b30c96cac5c3856f2/src/plugins/intel_cpu/src/nodes/subgraph.cpp#L691) since there are not implemented JIT emitters for `RegSpills`. What do you think?

Also it would be great if you create the ticket with task to support reg spills on ARM64 and mention this task in the comment near to disabling pass.